content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,155 @@
-==================================
-Configure Amazon Connector in Odoo
-==================================
+==============================
+Amazon Connector configuration
+==============================
 
-Register your Amazon account in Odoo
-====================================
+Odoo allows users to register an Amazon seller account in the database, but the user **must** have
+a paid Amazon Seller account prior to completing the configuration.
+
+To learn how to set up a paid Seller account on Amazon, log in to Amazon, and select *Start a
+Selling Account* from the *Account & Lists* drop-down menu, located in the header. Once that process
+is complete, proceed to follow the instructions below to register and link that Amazon Seller
+account in Odoo.
+
+Register Amazon account in Odoo
+===============================
 
 .. _amazon/setup:
 
-To register your seller account in Odoo, navigate to :menuselection:`Sales --> Configuration
---> Settings --> Connectors --> Amazon Sync --> Amazon Accounts` and click on :guilabel:`CREATE`.
+To register a seller account in Odoo, navigate to :menuselection:`Sales app --> Configuration
+--> Settings --> Connectors section`, activate the :guilabel:`Amazon Sync` feature, and click
+:guilabel:`Save`.
+
+Then, return to :menuselection:`Sales app --> Configuration --> Settings --> Connectors section`,
+and click on the :guilabel:`Amazon Accounts` link under the :guilabel:`Amazon Sync` setting.
+
+.. image:: setup/amazon-accounts-link-setting.png
+   :align: center
+   :alt: The Amazon Accounts link beneath the Amazon Synch settings in Odoo Sales.
+
+Doing so reveals a separate :guilabel:`Amazon Accounts` page. From here, click :guilabel:`New` to
+create, and subsequently link, a new Amazon account.
+
+On the blank :guilabel:`Amazon Account` form page, start by choosing a name for the account (e.g.
+`American Marketplace`). Then, in the :guilabel:`Credentials` tab, select the marketplace on which
+the seller account was initially created from the :guilabel:`Home Marketplace` drop-down menu.
+
+.. image:: setup/amazon-accounts-form-page.png
+   :align: center
+   :alt: A typical Amazon Account form page in the Odoo Sales application.
+
+.. note::
+   After validation of credentials, the marketplaces to which this account has access are
+   synchronized, and automatically made available.
```

Comment:
Isn't it strange to have this note before explaining how to validate the credentials? Especially since it's clearly written after, outside a note (L61-62) :thinking:

Reply:
Good call. I'll remove this note right away. Thanks for pointing this out. 👍